src/resource/app/dto/index.ts

```diff
@@ -0,0 +1 @@
+export { ContactFormInput } from '../../app/dto/contact-form.input';
```

Comment:
Maybe just from `./contact-form.input`? as they're under the same folder

Reply:
resolved